src/components/mobile-menu-container/index.tsx

```diff
@@ -56,23 +57,26 @@ const MobileAuthenticationControls = () => {
 	let content
 	if (showUnauthenticatedUI) {
 		content = (
-			<>
-				<Button
-					icon={<IconSignIn16 />}
-					iconPosition="trailing"
-					onClick={() => signIn()}
-					size="medium"
-					text="Sign In"
-				/>
-				<ButtonLink
-					color="secondary"
-					href="/sign-up"
-					icon={<IconUserPlus16 />}
-					iconPosition="trailing"
-					size="medium"
-					text="Sign Up"
-				/>
-			</>
+			<div className={s.unauthenticatedControls}>
+				<div className={s.unauthenticatedControlButtons}>
+					<Button
+						iconPosition="trailing"
+						onClick={() => signIn()}
+						size="medium"
+						text="Sign In"
+					/>
+					<ButtonLink
+						color="secondary"
+						href="/sign-up"
+						iconPosition="trailing"
+						size="medium"
+						text="Sign Up"
+					/>
```

Comment:
`iconPosition` props can be removed now that no `icon` is passed! 😊

```suggestion
					<Button
						onClick={() => signIn()}
						size="medium"
						text="Sign In"
					/>
					<ButtonLink
						color="secondary"
						href="/sign-up"
						size="medium"
						text="Sign Up"
					/>
```

Reply:
Oh yes, thank you 😄